release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1727,10 +1732,55 @@ If these values are not defined in a failure domain configuration, for instance
 * Previously, on on the *Observe -> Alerting -> Silences* tab, the `creator` field was not autopopulated and was not designated as mandatory. This issue happened when the API made the field empty from {product-title} 4.15 and onwards. With this update, the field is marked as mandatory and populated with the current user for correct validation. (link:https://issues.redhat.com/browse/OCPBUGS-35048[*OCPBUGS-35048*])
 
 
+[discrete]
+[id="ocp-release-note-oc-mirror-bug-fixes_{context}"]
+==== oc-mirror
+
+* Previously, when using `oc-mirror --v2 delete --generate`command, the contents of the working-dir/cluster-resources folder were cleared. With this fix, the working-dir/cluster-resources folder is not cleaned when the delete feature is used. (link:https://issues.redhat.com/browse/OCPBUGS-48430[*OCPBUGS-48430*])
+
+* Previously, release images were signed using a SHA-1 key. On {op-system-base} 9 FIPS STIG-compliant machines, verification of release signatures using the old SHA-1 key failed due to security restrictions on weak keys. With this release, release images are signed using a new SHA-256 trusted key so that the release signatures no longer fail. (link:https://issues.redhat.com/browse/OCPBUGS-48314[*OCPBUGS-48314*])
+
+* Previously, when running oc-mirror v2 Technology Preview in mirror-to-disk mode within an disconnected environment, the tool attempted to download `graph.tar.gz` from `api.openshift.com`, causing the mirroring process to fail.
++
+With this fix, when the `UPDATE_URL_OVERRIDE` environment variable is set, oc-mirror first checks for the graph image in the oc-mirror cache. If the image is not found, oc-mirror skips it without failing, ensuring successful mirroring in air-gapped environments. (link:https://issues.redhat.com/browse/OCPBUGS-38037[*OCPBUGS-38037*])
```

Comment:
```suggestion
With this release, when the `UPDATE_URL_OVERRIDE` environment variable is set, oc-mirror first checks for the graph image in the `oc-mirror` cache. If the image is not found, oc-mirror skips it without failing, ensuring successful mirroring in air-gapped environments. (link:https://issues.redhat.com/browse/OCPBUGS-38037[*OCPBUGS-38037*])
```
Assuming the cache is called `oc-mirror`, otherwise, disregard